### DISCUSSION TITLE
Notice timezones

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#637] [Q&A] [answered]
I started here the discussion: https://github.com/OP-TED/eForms-SDK/discussions/226
Thanks there the reply but the question is live.

The problem again:
The next DST is on October 29th.

We have a notice what we submitted on October 20th (before DST).
In this notice we refer to
1. date: October 26th ((before DST))
2. date: November 10th (after DST)

So what time zone we have to use in this case?
* The timezone of the date of notice submitting (2023-10-26+02:00 and 2023-11-10+02:00)?
* The timezone of the date itself (2023-10-26+02:00 and 2023-11-10+01:00)?

How we have to define the timezone in that situation?

We had problem with timezones in spring's DST that is why we now use "+00" at tiem fields and it is working but we know that is not the good way.

**Comment by YvesJo** [comment, marked as answer]
Hi,
There aren't much restrictions on timezones, just make sure that when a date is associated to a time, both share a same timezone. 
You may expect users to refer to their local time and it is certainly not desired to have them start calculating in order to express date and time in another timezone than their local one. In my opinion, the best approach would be to use the timezone that will be active on the specific day & time (i.e. your 2nd proposal).
I mark this discussion with a "help-wanted" tag to invite others to share their experience and I hope this will help you with your decision making process.
KR
